lib/jormungandr/test/data/jormungandr/stake_pools/create-stake-pool.sh

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env bash
+set -euo pipefail
+
+# This script (re)generates the required keys and certificates to setup a
+# stake-pool.
+#
+# Generated keys and certificates are checked in to git, and can manually be
+# inlined in genesis.yaml.
+#
+# The files are stored in a folder such that multiple stake-pools can be setup
+# in a convenient way.
+#
+# Usage:
+# Run in the stake_pools directory, where owner.prv exists.
+#
+# To create a stake pool in a folder "a":
+# .create-stake-pool.sh a
+
+echo "creating certificates for a new stake pool in the folder $1"
+mkdir $1
+cd $1
+
+jcli key generate --type=Curve25519_2HashDH > stake_pool_vrf.prv
```

Comment:
I bet it does not matter here, but in `$ jcli key generate --help` it is `curve25519_2hashdh`